### PR TITLE
Add version Nos for Maven plugins to the POM

### PR DESCRIPTION
**Pull request by imunro**
Addresses build maven build warnings and also a minor bug spotted during testing w/FLIMfit.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -24,7 +24,7 @@
         <dependency>
           <groupId>ome</groupId>
           <artifactId>bioformats_package</artifactId>
-          <version>5.1.6</version>
+          <version>5.1.8</version>
```
Comment (joshmoore) [inline pom.xml]: This could likely be omitted as in the blitz block above. The version will be taken from pom-omero-client.

**Comment by joshmoore** [comment]
Travis is green and the change makes sense (barring one comment). Should this be released when ready?

File: pom.xml (hunk 1 above)
Comment (imunro) [inline pom.xml]: I tried this by Deleting line 27 and could't build locally.

**Comment by imunro** [comment]
Up to you Josh. Hopefully there won't be further changes but we can wait until I've done more testing with the upcoming FLIMfit if it's easier.

File: pom.xml (hunk 1 above)
Comment (joshmoore) [inline pom.xml]: Sorry, the problem with just deleting them is that `formats-gpl` and `formats-bsd` are listed in `pom-omero-client` and not `bioformats_package`. I'm not sure what you're looking to pull in by using `bioformats_package` but perhaps better to keep it that way unless you can do the same via `formats-gpl`.

**Comment by joshmoore** [comment]
Merging. We can discuss `bioformats_package` in a follow-up.

**Comment by joshmoore** [comment]
https://bintray.com/joshmoore/maven/ome-ui-utils/0.1.4/view

**Comment by imunro** [comment]
Thanks Josh. I'd like to talk about Balaji's changes as well at some point.

**Comment by joshmoore** [comment]
mit mir?? Looking forward to learning ;)

**Comment by imunro** [comment]
yep!

**Pull request merged**